src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -100,11 +100,11 @@ After verifying that the sitemaps are built, you can add them to your site's `<h
 ```html ins={3}
 // src/layouts/Layout.astro
 <head>
-  <link rel="sitemap" href="/sitemap-index.xml">
+  <link rel="sitemap" href="/sitemap-index.xml" />
 </head>
 ```
 
-```yaml ins={4} title="public/robots.txt"
+```txt ins={4} title="public/robots.txt"
```

Comment:
We can perhaps exclude the language here. (Need to see if Prettier accepts that in the Astro repo 😅)

````suggestion
``` ins={4} title="public/robots.txt"
````